doc/nrf/known_issues.rst

```diff
@@ -155,7 +155,7 @@ nRF5
 nRF5340
 =======
 
-.. rst-class:: v1-4-99-dev1
+.. rst-class:: v1-5-0
 
 802.15.4 Service Layer (SL) library support for the nRF53
   The nRF53 support in the 802.15.4 Service Layer (SL) library is available only in the open-source version of the library.
```

Comment:
```suggestion
  The binary variant of the  802.15.4 Service Layer (SL) library for nRF53  does not support such features as synchronization of TIMER with RTC, timestamping of received frames. Because of that such 802.15.4 features as delayed transmission or delayed reception are note available for nRF53.
```

Reply:
Added with reference to the provided JIRA issue.